### ISSUE TITLE
Getting exception trying to call function using InvokeFunctionExample

### ISSUE THREAD

**Issue by c094728** [oracle/oci-java-sdk#372]
I'm trying to call a function using the code from InvokeFunctionExample.java and am getting an exception 

javax.ws.rs.ProcessingException: RESTEASY004655: Unable to invoke request: javax.ws.rs.ProcessingException: RESTEASY003215: could not find writer for content-type application/octet-stream type: com.oracle.bmc.io.internal.WrappedByteArrayInputStream
Any idea what is causing this?  stack trace is attached

[stacktrace.txt](https://github.com/oracle/oci-java-sdk/files/8241899/stacktrace.txt)

**Comment by jodoglevy** [comment]
@c094728 did the above resolve your issue?

**Comment by c094728** [comment]
@jodoglevy 
Yes, I was able to resolve it initially by excluding org.jboss.resteasy from oci-java-sdk-functions:2.19.0 dependency.  But then I had issues when I merged the stand-alone example into a spring boot application.  To resolve that I performed the following steps

- Update spring boot version from 1.5.4 to 2.6.4 (latest) That also required updating springfox-swagger-ui:2.8.0 to springfox-boot-starter:3.0.0 This updated a lot of other dependencies as well.

- We were using Powermock for some of the unit tests but that had compatibility problems with these changes so I removed powermock library and replaces with mockito-inline:4.3.1 (Latest) which supports mocking static methods. Older version of mockito could not mock static methods which is the whole reason powermock was being used.

- I updated all of the unit tests to use the new way of mocking static methods and got them all to run 
- Now the application was throwing an exception down in the springfox library and this prevented start up. I found a work-around for that here.   https://github.com/springfox/springfox/issues/3462
That work-around required importing spring-boot-starter-actuator library. 
Some of the comments in that blog say that springfox has been abandoned and we should migrate to spring-docs for our swagger-ui. That would require a lot more refactoring so I did not pursue it

After I did all this library updating I no longer needed to exclude Resteasy.  I'm not sure of the reason.

But bottom line is yes I did get it to run,  thank you for your help

I'm only using the part of the example that invokes a function.   My function was deployed using fn command line tool.    I see there is also code in that example that will deploy a function from an image when an application or function does not already exist on the server (i.e. true serverless).  and also remove all traces of it when done.   What is the use case for that?   Is that mainly for functions that are called infrequently? How much would that slow down the process of calling a function?
